### DISCUSSION TITLE
Missing button in Operator Decision Tree

### DISCUSSION THREAD

**Discussion by DavidePassafaro** [ReactiveX/rxjs#7242] [Report issues other than bug]
Hi everyone,
when accessing the [Operator Decision Tree](https://rxjs.dev/operator-decision-tree) I found something is missing.

This are the steps to reproduce:
1. Click on `I have one existing Observable, and` button
2. Missing button `I want to allow some values to pass` replaced with `You want the function of.` result.

![Screenshot](https://user-images.githubusercontent.com/29404081/230145331-ee074a53-8c3f-4a73-b335-6137277bcd85.PNG)

Seems this is related to the wrong generation of a json file: `/generated/docs/app/decision-tree-data.json`

Apparently the `8d34` property related to the first choise has an array of sub-choise, and at index 3 there is the id of the `of` operator result `6c39` instead of the correct one related to the sub-choise.

Not sure how to help more, that part of the code is a little bit tricky for me now but I'll spend some more time trying to understand what is causing the error.
Maybe it's just a wrong generation of the file and running again the script the file will be correctly restored.

**Comment by jakovljevic-mladen** [comment]
Hi @DavidePassafaro, thanks for reporting this issue.

It really seems that `/generated/docs/app/decision-tree-data.json` file is having some issues. Without further investigation, since I have an open PR for updating our docs app to Angular 15 which can be found [here](https://github.com/ReactiveX/rxjs/pull/7238), I used the code from that branch and I couldn't reproduce it. BTW, I found another issue which I fixed (related to StackBlitz).

Therefore, I hope that merging that PR will solve the issue. Once merged, I will publish the latest version online and we'll review.

**Comment by jakovljevic-mladen** [comment]
With the latest release of the docs app, this issue seems to be fixed. Thanks for reporting and feel free to file a new issue if you find one.
